apps/web/src/hooks/useLockedEndNotification.tsx

```diff
@@ -10,7 +10,6 @@ import atomWithStorageWithErrorCatch from 'utils/atomWithStorageWithErrorCatch'
 import { useUserCakeLockStatus } from './useUserCakeLockStatus'
 
 const storage = createJSONStorage(() => sessionStorage)
```

Comment:
I feel like making a copy of atom with storage try catch is a bit smelly.

Actually if we really need to try catch on localstorage we can use this pattern to assign a try catch setitem verion of localstorage in createjsonstorage